pkg/uploadserver/uploadserver_test.go

```diff
@@ -35,28 +38,28 @@ import (
 	"kubevirt.io/containerized-data-importer/pkg/importer"
 	"kubevirt.io/containerized-data-importer/pkg/util/cert"
 	"kubevirt.io/containerized-data-importer/pkg/util/cert/triple"
+	"kubevirt.io/containerized-data-importer/tests/reporters"
 )
 
+func TestUploadServer(t *testing.T) {
```

Comment:
In general we try to have a single 'suite' file that has the 'Testing.T' in it. We name it: _suite_test.go with the package here being apiserver. It also allows us to set things like logger in the before suite. A decent example is in pkg/controller/controller_suite_test.go